src/components/Rocket/Rocket.jsx

```diff
@@ -0,0 +1,29 @@
+import { useState, useEffect } from "react";
+import activeIcon from '../../assets/rocketActive.svg';
+import destroyedIcon from '../../assets/rocketDeactivate.svg';
+import unknownIcon from '../../assets/rocketUnknown.svg';
+import styles from './rocket.module.css';
```

Comment:
```suggestion
import { useState, useEffect } from "react";

import styles from './rocket.module.css';

import activeIcon from '../../assets/rocketActive.svg';
import destroyedIcon from '../../assets/rocketDeactivate.svg';
import unknownIcon from '../../assets/rocketUnknown.svg';

```
un par de enters solo fara facilitar la lectura